pipeline/id_minter/id_minter_works/src/main/scala/uk/ac/wellcome/platform/id_minter_works/Main.scala

```diff
@@ -32,10 +40,14 @@ object Main extends WellcomeTypesafeApp {
       )
     )
 
+    val elasticClient = ElasticBuilder.buildElasticClient(config)
+    val index = Index(config.requireString("es.index"))
```

Comment:
Minor: long term, it'd be nice if these flags reflected the type of index, e.g.

```suggestion
    val index = Index(config.requireString("es.denormalisedWorksIndex"))
```

Reply:
It's not the `denormalisedWorksIndex` that is read from yet, that will be once https://github.com/wellcomecollection/catalogue/pull/950 is merged